hack/run-wsu-ci-e2e-test.sh

```diff
@@ -3,19 +3,25 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
+WMCO_ROOT=$(pwd)
 TEST_DIR=$WMCO_ROOT/tools/ansible/tasks/wsu/test/e2e
 
-#TODO: Currently exiting early as the CI operator job which runs this does not have pip installed,
-#      which is a requirement to run this script
-exit 0
+# Make gopath if doesnt exist
+mkdir -p $GOPATH
+
+# If current user cannot be found add it to /etc/passwd. This is the case when this is run by an OpenShift cluster
```

Comment:
I think you're adding a temp user right?

Reply:
Comment about the assumption like `OpenShift assigns random user to container` as this is something that'll be helpful for all the people using ansible playbook within OpenShift.

Reply:
Changed